src/docs/development/add-to-app/android/project-setup.md

```diff
@@ -214,6 +214,13 @@ dependencies {
 }
 ```
 
+{{site.alert.important}}
```

Comment:
This is a minor nit, and don't fix it unless you have to make other changes, but notes should be indented on each line by 2 spaces. We also no longer use that format for URLs, but I can fix that on a future pass. 

Otherwise, LGTM, but will wait for @ blasten to review.